smtk/mesh/testing/cxx/UnitTestCellTypes.cxx

```diff
@@ -13,7 +13,7 @@
 
 #include "smtk/mesh/testing/cxx/helpers.h"
 
-#ifndef _MSC_VER
+#ifdef __GNUC__
```

Comment:
Nope, clang also recognizes GCC warning pragmas. This should stay as `#ifndef _MSC_VER`.